xayn-ai/src/data/mod.rs

```diff
@@ -87,6 +135,38 @@ impl UserInterests {
             negative: Vec::new(),
         }
     }
+
+    /// Moves all user interests of `other` into `Self`.
+    pub(crate) fn append(&mut self, mut other: Self) {
```

Comment:
I think that this block of new functions should belong to the `sync` module but I don't have a strong opinion about it.

Reply:
sounds sensible. now moved to a separate impl block in `sync`.